Domain/Do-Svyazi.User.Domain/Chats/Channel.cs

```diff
@@ -33,30 +34,59 @@ public class Channel : Chat
         CanDeleteChat = ActionOption.Disabled,
     };
 
-    public Channel(string name, string description)
+    public Channel(MessengerUser user, string name, string description)
         : base(name, description)
     {
+        var admin = new ChatUser
+        {
+            Role = _baseAdminRole,
+            ChatId = Id,
+            User = user,
+        };
+
+        Users.Add(admin);
         BaseAdminRole = _baseAdminRole;
         BaseUserRole = _baseUserRole;
     }
 
     public override void AddUser(ChatUser chatUser)
```

Comment:
По-моему, в чат приходит MessengerUser, и уже при добавлении получается ChatUser
```suggestion
    public override void AddUser(MessengerUser user)
```

Аналогично в остальных методах